fury/geometry.py

```diff
@@ -299,3 +299,125 @@ def create_image(image_input, material, **kwargs):
         Geometry(grid=Texture(image.astype(np.float32), dim=2)), material=material
     )
     return image
+
+
+def generate_ring(
```

Comment:
I am not sure where to put these functions, the file name resonated so added it here. Suggest me where will be a good place to add these.

Why I skipped
primitive.py -> cur function doesnt hold up the format of returning only the vertices and triangles
actor>planer.py -> cur func only generates a geometry not an actor
ui>helper.py -> cur func is not a ui related functionality

Reply:
Sure will add it into the primitive. I was referring the pygfx implementation of geometry so I added it that way.